### PR TITLE
Reference docs updates

### PR DESCRIPTION
**Pull request by jasonmadigan**
Some updates to the reference docs, prior to inclusion as a component on our docs website.

See https://github.com/Kuadrant/kuadrantctl/issues/52

### CODE REVIEW

**Comment by R-Lawton** [comment]
👀

File: README.md [hunk 1]
```diff
@@ -1,23 +1,151 @@
...
 
 ## Installing
-Use `go install` to install the latest version of the library. This command will install the `kuadrantctl` binary executable in `$GOBIN` (defaulting to `$GOPATH/bin`).
 
-```
+`kuadrantctl` can be installed either by downloading pre-compiled binaries or by compiling from source. For most users, downloading the binary is the easiest and recommended method.
+
+### Installing Pre-compiled Binaries
+
+1. Download the latest binary for your platform from the [`kuadrantctl` Releases](https://github.com/Kuadrant/kuadrantctl/releases) page.
```
Comment (R-Lawton) [inline README.md:12]: Putting the kuadrantctl in quotes causes the link to split into 2 separate links both work just might cause weird things on website

File: README.md [hunk 2]
```diff
@@ -1,23 +1,151 @@
...
+
+#### Generating Gateway API HTTPRoute Objects
+
+- Generates [Gateway API HTTPRoute](https://gateway-api.sigs.k8s.io/v1alpha2/guides/http-routing/) objects from an OpenAPI Specification (OAS) 3.x.
+- Supports reading from a file, URL, or stdin.
+- Example usages and more information can be found in the [detailed guide](doc/generate-gateway-api-httproute.md).
+
+#### Generating Kuadrant AuthPolicy Objects
+
+- Generates [Kuadrant AuthPolicy](https://github.com/Kuadrant/kuadrant-operator/blob/v0.4.1/doc/auth.md) objects for managing API authentication.
```
Comment (R-Lawton) [inline README.md:129]: why version 0.4.1 for the absolute links?

Comment (jasonmadigan) [reply README.md:129]: Thought I'd leave these links at 0.4.1 right now as that's what `kuadrantctl install` installs right now (until https://github.com/Kuadrant/kuadrantctl/issues/56 is fixed)

File: README.md (hunk 1 above)
Comment (jasonmadigan) [reply README.md:12]: Screenshot 2024-03-01 at 11 46 29

Seems to render OK, although I'll go through the rest for rendering sanity now that you mention it.

**Comment by jasonmadigan** [comment]
Copy of mkdocs rendered content in this draft PR for docs:

https://github.com/Kuadrant/docs.kuadrant.io/pull/77

File: README.md (hunk 1 above)
Comment (R-Lawton) [reply README.md:12]: oh deadly

File: README.md (hunk 2 above)
Comment (R-Lawton) [reply README.md:129]: ah ok makes sense maybe a note so we don't forget?

**Review by david-martin — APPROVED** [review]
/lgtm

Out of interest, are the command overview tables generated  from the cli or the help output?
Thinking how they'd be most easily updated in future

**Review by eguzki — APPROVED** [review]
:1st_place_medal:

**Comment by R-Lawton** [comment]
lgtm

**Pull request merged**